### PR TITLE
Home Assistant

### PR DESCRIPTION
**Pull request by muuuh**

### CODE REVIEW

File: Weather/homeassistant.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		sensor1_humidity = item
+	if item['entity_id'] == 'switch.plug_switch':
+		plug_switch = item
+
+from datetime import datetime
+
+sensor1_temperature_value = (sensor1_temperature_temperature['state'] + " " + sensor1_temperature_temperature['attributes']['unit_of_measurement']).encode('utf8')
+sensor1_temperature_changed_minutes = str((datetime.utcnow() - datetime.strptime(sensor1_temperature_temperature['last_changed'], '%Y-%m-%dT%H:%M:%S.%f+00:00')).seconds / 60).encode('utf8')
+
+sensor1_humidity_value = (sensor1_humidity_temperature['state'] + " " + sensor1_humidity_temperature['attributes']['unit_of_measurement']).encode('utf8')
```
Comment (tresni) [inline Weather/homeassistant.py]: Is `sensor1_humidity_temperature` suppose to be `sensor1_humidity` from L59?

Comment (sprak3000) [reply Weather/homeassistant.py:68]: @muuuh 

Would it be possible to answer this?

File: Weather/homeassistant.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	if item['entity_id'] == 'sensor.sensor1_temperature':
+		sensor1_temperature = item
+	if item['entity_id'] == 'sensor.sensor1_humidity':
+		sensor1_humidity = item
+	if item['entity_id'] == 'switch.plug_switch':
+		plug_switch = item
+
+from datetime import datetime
+
+sensor1_temperature_value = (sensor1_temperature_temperature['state'] + " " + sensor1_temperature_temperature['attributes']['unit_of_measurement']).encode('utf8')
```
[the remark this answers is not in the document]
Comment (sprak3000) [reply Weather/homeassistant.py:65]: @muuuh 

Would it be possible to answer this?

**Comment by sprak3000** [comment]
@muuuh 

Please add a `<xbar.image>` metadata tag linking to a screenshot of the plugin in action and run `chmod +x` on the script file.

**Review by sprak3000 — APPROVED** [review]

**Pull request merged**